tests/tracker_ignored_ddl/run.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -eu
+
+cur=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+source $cur/../_utils/test_prepare
+WORK_DIR=$TEST_DIR/$TEST_NAME
+
+function run() {
+	run_sql_file $cur/data/db.prepare.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+
+	run_dm_master $WORK_DIR/master $MASTER_PORT $cur/conf/dm-master.toml
+	check_rpc_alive $cur/../bin/check_master_online 127.0.0.1:$MASTER_PORT
+	run_dm_worker $WORK_DIR/worker1 $WORKER1_PORT $cur/conf/dm-worker1.toml
+	check_rpc_alive $cur/../bin/check_worker_online 127.0.0.1:$WORKER1_PORT
+
+	# operate mysql config to worker
+	cp $cur/conf/source1.yaml $WORK_DIR/source1.yaml
+	dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
+
+	# start DM task only
+	dmctl_start_task_standalone $cur/conf/dm-task.yaml
+	# use sync_diff_inspector to check full dump loader
+	check_sync_diff $WORK_DIR $cur/conf/diff_config.toml
+
+	run_sql_file $cur/data/db.increment1.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_sql_tidb "show create table $TEST_NAME.t1;"
+	check_not_contains "ignore_1"
+	echo "increment1 check success"
+
+	run_sql_file $cur/data/db.increment2.sql $MYSQL_HOST1 $MYSQL_PORT1 $MYSQL_PASSWORD1
+	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"query-status test" \
```

Comment:
I'm curious if this solves the problem at all
It just changes the error message from `count unmatch errors` to `Unknown column`.

Maybe we can go a step further and keep the sync task from being interrupted?

If a column is added upstream (which is ignored), the DM generated DML should also ignore that column

If a column is subtracted upstream (which is ignored), the DM generated DML should also take the default value of that column

also cc @lance6716  @lichunzhu

Reply:
also cc @glorv

Reply:
there's already a lot of content in this PR, so maybe we can implement this in next PR

Reply:
> If a column is subtracted upstream (which is ignored), the DM generated DML should also take the default value of that column
@Ehco1996  We don't support this kind of operation and don't have plan to support this kind of operation at least now. Since this kind of operations will lead to data inconsistency.  And #1717 is not related to it either, the problem is that even if the down stream is compatible with upstream, e.g. user manually changed the schema, dm still reported an error which is not acceptable. 

So I think this PR is ok.

Reply:
I think we can add the alter table ddl in the script before executing the incremental phase, then the task should finish successfully.